packages/web-client/app/utils/web3-strategies/layer2-chain.ts

```diff
@@ -123,7 +123,10 @@ export default abstract class Layer2ChainWeb3Strategy
   async fetchIssuePrepaidCardMinValues() {
     this.issuePrepaidCardSpendMinValue = Math.min(...faceValueOptions);
     this.issuePrepaidCardDaiMinValue = new BN(
-      await this.convertFromSpend('DAI', this.issuePrepaidCardSpendMinValue)
+      await this.convertFromSpend(
+        this.bridgedDaiTokenSymbol,
```

Comment:
Are you sure that the Oracle from the SDK method accepts 'DAI.CPXD' and 
'CARD.CPXD'?

Reply:
No, it doesn't. This is for the mapping in `TokenContractInfo` which resolves to `daiCpxd` and `cardCpxd`.

Reply:
Leaving it at `DAI` would work for Sokol but break in xDai

Reply:
OK, let's go with it.